include/leds.h

```diff
@@ -0,0 +1,30 @@
+#ifndef LEDS_H
+#define LEDS_H
+
+#include "board.h"
+
+enum oca_led_select
+{
+	oca_led_act = GPIOB_LED_ACT,
+	oca_led_stat = GPIOB_LED_STAT,
+	oca_led_err = GPIOB_LED_ERR
+};
+
+enum oca_led_state
+{
+	oca_led_off = 0,
+	oca_led_on,
+	oca_led_blink_250,
+	oca_led_blink_500,
+	oca_led_blink_1000
+};
+
+void oca_init_leds(void);
+
+void oca_set_led(enum oca_led_select led, enum oca_led_state state);
+void oca_toggle_led(enum oca_led_select led);
+
+void oca_update_leds(void); //call every 250ms
```

Comment:
almost, in C programming you could fake namespaces as in C++ classes with <project>_<file>_<function>. So for oca led update write: 
`oca_led_update` or `oca_led_toggle`. This is easy when you mixup alot of code from other projects and places. Let me know what you think.